spcplay.exe/spcplay.dpr

```diff
@@ -3510,11 +3511,11 @@ const
         ('&Normal', 'OLD &Sound Blaster Card', 'OLD &ZSNES, Snes9x'));
     STR_MENU_SETUP_OPTION_SUB: array[0..1] of array[0..MENU_SETUP_OPTION_SIZE - 1] of pchar = (
         ('���@���[�p�X �t�B���^(&L)', '���@�G�R�[/FIR ����(&M)', NULLPOINTER, '&BASS BOOST', '�ߋ��� &ADPCM �f�R�[�_', '�t�ʑ��T���E���h����(&S)',
-         '���E���](&R)', '�G���x���[�v���x��𓯊�(&Y)', NULLPOINTER, '�T���E���h����(&U)', '�G�R�[����(&E)', '&FIR �t�B���^����',
-         '�s�b�` ���W�����[�V��������(&P)', '�s�b�` �x���h����(&I)', '�s�b�` ���~�b�g����(&T)', '�G���x���[�v����(&V)', '�m�C�Y�����w�薳��(&N)'),
+         '���E���](&R)', '�G���x���[�v���x��𓯊�(&Y)', NULLPOINTER, '�T���E���h����(&U)', '���C������(&M)', '�G�R�[����(&E)',
+         '&FIR �t�B���^����', '�s�b�` ���W�����[�V��������(&P)', '�s�b�` �x���h����(&I)', '�s�b�` ���~�b�g����(&T)', '�G���x���[�v����(&V)', '�m�C�Y�����w�薳��(&N)'),
         ('SNES &Low-Pass Filter', 'SNES Echo/FIR &Method', NULLPOINTER, '&BASS BOOST', 'Old &ADPCM Decoder', 'Opposite-Phase &Surround',

```

Comment:
> Unfortunately, the shortcut is mapped to `M`, which is the same as `SNES Echo/FIR &Method`.

How about changing the shortcut key for "SNES Echo/FIR &Method" to "C"?
I agree that "Main" should be set as the shortcut key to "M".

By the way, the Japanese translation is correct!
The placement position of the "Disable Main" item is also great.

Reply:
No problem at all. I've made and tested the requested changes. Would you prefer that I force-push an amended commit or create a new one?

Reply:
Either is fine.
Ultimately, all commits in this pull request will be merged into one commit by squash merge.